db/migrate/20210811194501_remove_deployment_from_authentications.rb

```diff
@@ -0,0 +1,69 @@
+class RemoveDeploymentFromAuthentications < ActiveRecord::Migration[6.0]
+  def up
+    change_table "authentications" do |t|
+      t.remove :challenge
+      t.remove :login
+      ## keeping: t.remove :public_key
```

Comment:
Probably should remove this comment?

Reply:
I liked having it in here but am more than happy to remove